### PR TITLE
Style: Home nav & menu link background when on home page

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy 

topLevelMap is compiled using navItem's `id` rather than `href` given they have same uniqueness.

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
Why not href, rather than adding a new property?

**Comment by andygout** [comment]
All other items already had an `id`; they only needed to be added to the two home links and portfolio. Seemed fitting to use `id`s as the values that would then be used as the id values in the topLevelMap.

**Comment by ironsidevsquincy** [comment]
The other ids are actually capi v1 ids for those items (which should probably be deprecated anyway, and moved over to v2). I think it's a bit confusing (and dangerous) to mix them with ones we've just made up

File: src/server/lib/data.js [hunk 1]
```diff
@@ -32,12 +32,12 @@ const clone = obj => { return JSON.parse(JSON.stringify(obj)); };
 const config = require('../../../config/nav');
 
 // top level contains both uk and us, which gets filtered by edition on each request
-const topLevelSections = ['home', 'world', 'uk', 'us', 'companies', 'markets', 'opinion', 'life & arts', 'portfolio'];
+const topLevelSections = ['/uk', '/international', '/stream/sectionsId/MQ==-U2VjdGlvbnM=', '/stream/sectionsId/Ng==-U2VjdGlvbnM=', '/stream/sectionsId/Mg==-U2VjdGlvbnM=', '/stream/sectionsId/Mjk=-U2VjdGlvbnM=', '/stream/sectionsId/NzE=-U2VjdGlvbnM=', '/stream/sectionsId/MTE2-U2VjdGlvbnM=', '/stream/sectionsId/MTQ4-U2VjdGlvbnM=', '/myft/portfolio'];
```
Comment (ironsidevsquincy) [inline src/server/lib/data.js]: bit long this line now. generally rule is keep line length at 120 characters or less

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**